.travis.yml

```diff
@@ -5,6 +5,7 @@ services:
 before_install:
   - mkdir -p _output/
   - sudo apt-get install -y git
+  - sudo apt-get install libpcap-dev
```

Comment:
i think the best way would be to add bash script with following content, and execute it inside container

```
apt-get update
apt-get install -y libpcap-dev
go build -o _output/ipmanager ./cmd/ipmanager/
```